scanners/screenshooter/integration-tests/screenshooter.test.js

```diff
@@ -0,0 +1,19 @@
+// SPDX-FileCopyrightText: the secureCodeBox authors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+const {scan} = require("../../helpers");
+
+test(
+  "make screenshot of nginx demo target",
+  async () => {
+    const { categories } = await scan(
+      "demo-target-screenshot",
+      "screenshooter",
+      ["http://nginx.demo-targets.svc"],
+      60 * 4
+    );
+
+    expect(categories).toBeDefined();
```

Comment:
I think a proper expect to check if the actual screenshot category here would be better.
It could be defined but empty-